### DISCUSSION TITLE
How to enable sorting on display columns?

### DISCUSSION THREAD

**Discussion by giopetris** [TanStack/table#5080] [Q&A]
My pagination, sorting and filtering are all backend. And the func `column.getCanSort()` always returns false for `display` columns, I have to use `accessor` columns to enable it. But, in my case, since sorting is done on the backend, the `accessor` is not necessary. 

Is there some way to have sorting enabled for `display` columns?

**Comment by rixth** [comment]
@giopetris Did you find a workaround for this?
